README.md

```diff
@@ -8,4 +8,4 @@ Please find the sample that fits your use-case from the table below.
 |--------|-------------|----------|
 | [Browser Sign In](/browser-sign-in) | An application that integrates [OktaAuth SDK](https://github.com/okta/okta-sdk-appauth-ios) to perform authorization. | Authorization performed by redirecting from native app to browser. |
 | [Custom Sign In](/custom-sign-in) | An application that integrates [OktaAuth Swift SDK](https://github.com/okta/okta-auth-swift) to perform authorization. | An application that adopts native authorization to take control over authorization flow and/or provide custom UI. | 
-
+| [TOTP Generator](/totp-app) | A sample application that generates TOTP tokens. | An application that shows how developer can build their own Google Authenticator clone for their brand. | 
```

Comment:
I think we have to remove mentioning of `Google Authenticator` in readme. Probably better to mention `Okta Verify`?

@nbarbettini , thoughts?